ganttproject/src/main/java/net/sourceforge/ganttproject/task/TaskManagerImpl.java

```diff
@@ -355,15 +355,18 @@ public TaskBuilder newTaskBuilder() {
     return new TaskBuilder() {
       @Override
       public Task build() {
-
         if (myPrototype != null) {
           myId = myPrototype.getTaskID();
         }
         if (myId == null || myTaskMap.getTask(myId) != null) {
           myId = getAndIncrementId();
         }
 
-        TaskImpl task = new GanttTask("", CalendarFactory.createGanttCalendar(), 1, TaskManagerImpl.this, myId);
+        var startDate = myStartDate != null
```

Comment:
Should this part of the code be removed from `TaskManager.TaskBuilder.setupNewTask` then? The method is not used anywhere but here

Reply:
done, thanks